### PR TITLE
Table Removal 

### PR DESCRIPTION
**Pull request by rahulhazra97**
## Describe the Change

This PR contains edits and changes in layout and spacing for the Palette Onboarding Workflow under Getting Started on Spectro Cloud Docs.

## Review Changes

💻 [Preview URL](https://deploy-preview-1234--docs-spectrocloud.netlify.app/getting-started/onboarding-workflow)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1234#" title="Author self-approved">rahulhazra97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/02-getting-started/04-onboarding-workflow.md
Comment (karl-cardenas-coding) [inline content/docs/02-getting-started/04-onboarding-workflow.md:69]:
```suggestion
Palette requires the creation of a cluster profile before a workload cluster can be created. This is because [cluster profiles](/cluster-profiles) are
templates created with preconfigured layers that define the required dependencies, such as the Operating System (OS) and Kubernetes version for your cluster.  The cluster profile is a core component of Palette. You can learn more about cluster profiles by reviewing the [cluster profile](/cluster-profiles) reference page.
```

**Review by karl-cardenas-coding — APPROVED** [review]
Looks good, just a minor change.

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1234#pullrequestreview-1376311180" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1234#" title="Author self-approved">rahulhazra97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
